### PR TITLE
Update raw-events.md

### PR DESCRIPTION
**Pull request by zophiale**

### CODE REVIEW

File: coding-guides/raw-events.md [hunk 1]
```diff
@@ -43,23 +43,23 @@ client.on('raw', packet => {
     // We don't want this to run on unrelated packets
     if (!['MESSAGE_REACTION_ADD', 'MESSAGE_REACTION_REMOVE'].includes(packet.t)) return;
     // Grab the channel to check the message from
-    const channel = client.channels.get(packet.d.channel_id);
+    const channel = client.channels.cache.get(packet.d.channel_id);
     // There's no need to emit if the message is cached, because the event will fire anyway for that
-    if (channel.messages.has(packet.d.message_id)) return;
+    if (channel.messages.cache.has(packet.d.message_id)) return;
     // Since we have confirmed the message is not cached, let's fetch it
     channel.fetchMessage(packet.d.message_id).then(message => {
```
Comment (ghost) [inline coding-guides/raw-events.md:50]: fetchMessage is not a Channel method anymore. Moved into Managers.

**Comment by eslachance** [comment]
While this update is technically correct, v12 offers partials now, which means the entire idea of having to use raw events become moot in the case of reactions particularly. Instead of just updating the existing code I think we should review this page to use partials instead.

File: coding-guides/raw-events.md [hunk 2]
```diff
@@ -43,23 +43,23 @@ client.on('raw', packet => {
     // We don't want this to run on unrelated packets
     if (!['MESSAGE_REACTION_ADD', 'MESSAGE_REACTION_REMOVE'].includes(packet.t)) return;
```
Comment (eslachance) [inline coding-guides/raw-events.md:44]: Edit this to use partials instead to take advantage of the new v12 features

**Review by eslachance — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
This file has been updated completely in #102. This PR can be closed and a new one for rewrite to use partials could be forked from that PR. @eslachance

**Comment by eslachance** [comment]
No update from contributor after change requests and conversations.

**Pull request closed without merging**